webapp/i18n/en.json

```diff
@@ -818,6 +818,9 @@
   "admin.sql.keyTitle": "At Rest Encrypt Key:",
   "admin.sql.maxConnectionsDescription": "Maximum number of idle connections held open to the database.",
   "admin.sql.maxConnectionsExample": "E.g.: \"10\"",
+  "admin.sql.connectionTimeoutTitle": "Connection Timeout:",
+  "admin.sql.connectionTimeoutDescription": "The number of seconds a query can be executing for before being cancelled.",
```

Comment:
1. Should always be greater than 0, no maximum. They can set it really high if they never want it to timeout
2. The error will show in the server logs and the user will likely see an error on the client. What that looks like to the user could vary. For example it could be a blue bar error or an error on the form they were interacting with, etc.